.gitignore

```diff
@@ -89,4 +89,10 @@ fastlane/test_output
 
 iOSInjectionProject/
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Ignore project files as we generate them with xcodegen 
+#[https://github.com/yonaskolb/XcodeGen
```

Comment:
Is this opening square bracket intended?

Reply:
Nope! Thanks for pointing it out